gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateService.java

```diff
@@ -286,7 +287,15 @@ private void removeTokenState(final Set<String> tokenIds) {
     tokenExpirations.keySet().removeAll(tokenIds);
     maxTokenLifetimes.keySet().removeAll(tokenIds);
     metadataMap.keySet().removeAll(tokenIds);
-    log.removedTokenState(String.join(", ", tokenIds));
+    log.removedTokenState(String.join(", ", getDisplayableTokenIDsText(tokenIds)));
+  }
+
+  protected Set<String> getDisplayableTokenIDsText(final Set<String> tokenIds) {
```

Comment:
It feels better to have this method in `Tokens`.

Reply:
I agree.